app/controllers/channels/base_controller.rb

```diff
@@ -1,6 +1,11 @@
 class Channels::BaseController < ApplicationController
   skip_before_filter :set_locale
+  helper_method :channel_decorator
   before_filter do
     I18n.locale = 'pt'
   end
+
+  def channel_decorator
+    channel.decorator
```

Comment:
Pq eu to tenho um `helper_method :channel_decorator`, não ficaria estranho deixar só `helper_method: channel` ?